src/StoreKeeper/WooCommerce/B2C/Core.php

```diff
@@ -463,4 +474,91 @@ public function renderBootError(BootError $e)
             }
         }
     }
+
+    /**
+     * @param $settings
+     * @return mixed
+     */
+    public function ts_add_extra_fields_in_shipping_methods($settings) {
+        // Add a new setting for Minimum Amount
+        $new_settings = $settings;
+        $currency_code = get_woocommerce_currency_symbol();
+
+        $new_settings['min_amount'] = array(
+            'title'       => __('Minimum Cost (' . $currency_code .')', 'woocommerce'),
+            'type'        => 'text',
+            'class'       => 'currency-input2',
+            'placeholder' => '100.00', // Adjusted to reflect decimal format
+            'default'     => '100.00', // Default value as a decimal
+            'custom_attributes' => array(
+                'min' => 0,
+                'step' => '0.01' // Allow decimal steps in the input
+            ),
+        );
+        return $new_settings;
+    }
+
+    /**
+     * @return void
+     */
+    public function ts_display_shipping_min_amount_content() {
```

Comment:
```suggestion
    public function ts_display_shipping_min_amount_content(): void {
```

You use php 7.3 stadard features ( also change in other functions)